pkg/configOperator/configOperator.go

```diff
@@ -272,6 +272,9 @@ func (op ConfigOperator) handleIOSVariant(secret *BindingSecret) {
 // Deletes a configuration from the config secret and from the UPS server
 func (op ConfigOperator) handleDeleteVariant(secret *BindingSecret) {
 	appType := strings.ToLower(string(secret.Data["appType"]))
+	if appType == "" {
```

Comment:
I think it would be better to specifically check for `IOS` and `Android` here and dismiss all other secrets. Also please add a short comment describing the rationale of the check.